sgkit/tests/test_regenie.py

```diff
@@ -292,6 +292,7 @@ def test_regenie__glow_comparison(datadir: Path) -> None:
         check_simulation_result(datadir, config, run)
 
 
+@pytest.mark.xfail()  # See https://github.com/pystatgen/sgkit/pull/430#issuecomment-772868301
```

Comment:
`xfail` has a argument `reason`, that's probably a good place to link the issue. And btw moved the comment to separate issue at #456, and upstream issue is https://github.com/dask/dask/issues/7171